sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -161,7 +161,7 @@ Authentication with AAD requires some initial setup:
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-identity</artifactId>
-    <version>1.5.2</version>
+    <version>1.5.3</version>
```

Comment:
Do you also need to update the TA lib version [here ](https://github.com/Azure/azure-sdk-for-java/pull/29876/files#diff-c179d0b9fa6f6a38a9123c130241909b36537d8722a2d1b9de5bb5677928ccf8R64)